wgengine/netstack/netstack.go

```diff
@@ -670,7 +670,7 @@ func (ns *Impl) handleLocalPackets(p *packet.Parsed, t *tstun.Wrapper) filter.Re
 		return filter.DropSilently
 	}
 
-	// If it's not traffic to the service IP (e.g. magicDNS or TailFS) we don't
+	// If it's not traffic to the service IP (e.g. magicDNS or Taildrive) we don't
```

Comment:
Taildrive capitalized is good. Do this for the lowercase "drive" elsewhere?

Reply:
If we always refer to it as "Taildrive" in comments, capitalization of "drive" becomes moot.